lib/cloudregister/registerutils.py

```diff
@@ -2308,16 +2308,18 @@ def is_suma_instance():
     """Return True if there is a match for SUMA and SLE-Micro products,
     otherwise False."""
     products = glob.glob('/etc/products.d/*.prod')
-    num_matches = 0
+    # suse-manager-server changed to
+    # multi-linux-manager-server, check both (bsc#1243437)
     suma_prods = [
-        '/etc/products.d/suse-manager-server.prod',
-        '/etc/products.d/sle-micro.prod'
+        'manager-server.prod',
```

Comment:
we could compare `manager-server` instead of `manager-server.prod` 
in case the name gets more text before the extension in the future

Thoughts ?

Reply:
I think product names are not changing that often such that we should be good with sticking to matching against a part of the name that will not accidentally match with another product. I would even go that far and match the strings against `endswith(...)` instead of `a in b` but with the `.prod` in the matching text that's solid enough

Reply:
Personally I would prefer a list of complete product names and no substring matches. In terms of product names the list will not grow endlessly

Reply:
The reason for this is, your code becomes explicit and you can spare out a comment that explains that `manager-server` actually matches two products. So I would just go and maintain 

```
suma_prods = [
    'suse-manager-server.prod',
    'multi-linux-manager-server.prod',
    'sle-micro.prod'
```

Reply:
we can keep the minimum change in [fde052f](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/pull/249/commits/fde052ffc4bfd4dfda02a3a4e9b8ada24357c033)

or keep with this change and the full name in [fc31964](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/pull/249/commits/fc319640003e43077283b2097512276e7d05509c)